### DISCUSSION TITLE
Any PTFs needed for Debug/2 with SEP support?

### DISCUSSION THREAD

**Discussion by alanseiden** [codefori/vscode-ibmi#1978] [General]
The question came up during our Friday Code for i call.
The "Required PTFs" tab under Debug Version 2 says "Version 2 not yet released."
https://codefori.github.io/docs/developing/debug/#required-ptfs

**Comment by sebjulliand** [comment]
Indeed, a new PTF will be needed to add support for SEP to the debugger. As far as I know, it should be released in a few weeks.
